dev/docker/doris/start.sh

```diff
@@ -65,13 +68,56 @@ if [ "$fe_started" = false ]; then
   exit 1
 fi
 
+# check for be started
+be_started=false
+for i in {1..10}; do
+  sleep 5
+  # check localhost:8040/api/health return contains content '"status": "OK"' to see if the be is ready
+  url="localhost:8040/api/health"
+
+  echo "check be for the $i times"
+  response=$(curl --silent --request GET $url)
+
+  if echo "$response" | grep -q "\"status\": \"OK\""; then
+    be_started=true
+    echo "Doris be started successfully, response: $response"
+    break
+  else
+    echo "check failed, response: $response"
+  fi
+done
+
+if [ "$be_started" = false ]; then
+  echo "Doris be failed to start"
+
+  cat ${DORIS_BE_HOME}/log/*
+  exit 1
+fi
+
+
 # add the be to the fe
-mysql -h127.0.0.1 -P9030 -uroot -e "ALTER SYSTEM ADD BACKEND '${CONTAINER_IP}:9050'"
+be_added=false
+for i in {1..10}; do
+  echo "add Doris BE to FE for the $i times"
+
+  mysql -h127.0.0.1 -P9030 -uroot -e "ALTER SYSTEM ADD BACKEND '${CONTAINER_IP}:9050'"
+  if [ $? -ne 0 ]; then
+    echo "Failed to add the BE to the FE"
+  else
+    be_added=true
+    echo "Doris BE added to FE successfully"
+    break
+  fi
+
+  sleep 1
+done
+
+if [ "$be_added" = false ]; then
+  echo "Doris BE failed to add to FE"
+  cat ${DORIS_FE_HOME}/log/fe.* ${DORIS_BE_HOME}/log/*
 
-if [ $? -ne 0 ]; then
-  echo "Failed to add the BE to the FE"
   exit 1
 fi
 
 # persist the container
-tail -f ${DORIS_FE_HOME}/log/fe.log ${DORIS_BE_HOME}/log/be.log
+tail -f ${DORIS_FE_HOME}/log/fe.log ${DORIS_BE_HOME}/log/be.INFO
```

Comment:
be.INFO fe.log? Is it right?

Reply:
fe log filename is fe.log
be log filename is be.INFO


<img width="608" alt="image" src="https://github.com/datastrato/gravitino/assets/6739160/9ac30368-2c6d-42dd-b6aa-9ec50fbbee6a">
<img width="541" alt="image" src="https://github.com/datastrato/gravitino/assets/6739160/56a2d4f8-2b79-4fce-af26-48912cd9abd7">

Reply:
OK, but it's weird for a system.